table/block_based/block_based_table_reader_test.cc

```diff
@@ -250,6 +262,62 @@ class BlockBasedTableReaderTest
   uint32_t compression_dict_bytes_;
 };
 
+TEST_P(BlockBasedTableReaderTest, Get) {
+  Options options;
+  if (udt_enabled_) {
+    options.comparator = test::BytewiseComparatorWithU64TsWrapper();
+  }
+  options.persist_user_defined_timestamps = persist_udt_;
+  size_t ts_sz = options.comparator->timestamp_size();
+  std::vector<std::tuple<std::string, std::string>> kv =
+      BlockBasedTableReaderBaseTest::GenerateKVMap(
+          100 /* num_block */,
+          true /* mixed_with_human_readable_string_value */, ts_sz,
+          /*same_key_diff_ts=*/true);
```

Comment:
Question: would it make sense to make this a parameter of the test as well?

Reply:
Good point, it would be good to test both modes. I have updated `Get` to be parameterized with this too.